src/cli/OutputGTFSZipCommand.ts

```diff
@@ -29,8 +29,12 @@ export class OutputGTFSZipCommand implements CLICommand {
     // when node tells you it's finished writing a file, it's lying.
     setTimeout(() => {
       console.log("Writing " + filename);
-      execSync(`zip -j ${filename} ${argv[3]}/*.txt`);
+      execSync(`zip -j '${escapeShellArg(filename)}' ${argv[3]}/*.txt`);
     }, 1000);
   }
 
+}
+
+function escapeShellArg (arg) {
+  return `'${arg.replace(/'/g, `'\\''`)}'`;
```

Comment:
not sure I would call this escaping... more just quoting. I will check if there is a better in built function to use